test/vectors.test.ts

```diff
@@ -136,7 +139,7 @@ describeCryptoTests(({ provider, supportedSuites: supported }) => {
 
                     let info: Uint8Array | undefined = undefined
                     if (testVector.mode === Oprf.Mode.POPRF) {
-                        info = fromHex(vi.Info!)
+                        info = fromHex(vi.Info ?? '')
```

Comment:
nits: I prefer `!`. It's easier to read and debug
other option is an if statement

Reply:
once I discovered type assertions, these kind of checks become easier